demos/datepicker/datepicker-fr.js

```diff
@@ -6,7 +6,7 @@
 	if ( typeof define === "function" && define.amd ) {
 
 		// AMD. Register as an anonymous module.
-		define([ "../jquery.ui.datepicker" ], factory );
+		define([ "datepicker" ], factory );
```

Comment:
This is already in my pull on jqueryui.com i already need to special case these files. So it wont be any harder to special case them in this way and it would simplify the bootstrap and avoid this issue completely because they would be within the baseUrl already.